packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -164,29 +164,51 @@ const WorkflowStepStore = types
         // put is a MST method, not native to ES Map
         // the key is inferred from the identifier type of the target model
         const taskToStore = Object.assign({}, workflow.tasks[taskKey], { taskKey })
-        self.tasks.put(taskToStore)
+        try {
+          self.tasks.put(taskToStore)
+        }
+        catch (e) {
+          console.log(`${taskKey} ${taskToStore.type} is not a supported task type`)
+        }
       })
     }
 
     function convertWorkflowToUseSteps (workflow) {
       const taskKeys = Object.keys(workflow.tasks)
+      const { first_task } = workflow
+
+      function comboToStepTasks (task) {
```

Comment:
- This could be a pure function
- Having `taskKey` and `taskKeys` refer to different things in different scopes is a little confusing

Reply:
`task.tasks` (returned from the API) is an array of task keys. I'm not sure what I can do to make the naming clearer.

The tests are written for individual methods, so I've kept the changes here inside the `convertWorkflowToUseSteps` method.

Reply:
Looking at this again, the helper function is removing those combo tasks from the `taskKeys` array, so it can't run outside the scope of `convertWorkflowToUseSteps`.